### PR TITLE
Suggested tweaks

### PR DESCRIPTION
**Pull request by raoulsullivan**
These will require the form or translations are updated to deal with a move from percentages to integers for "proportion of sales" values.

### CODE REVIEW

File: sme_finance_application_schema/entity_v1 [hunk 1]
```diff
@@ -88,14 +85,13 @@
         },
         "overseas_revenue": {
             "title":"Overseas revenue",
-            "description":"The proportion of revenue the SME receives from sales outside the United Kingdom, expressed as a percentage",
+            "description":"The monthly revenue the SME receives from sales outside the United Kingdom, in GBP",
```
Comment (nesaro) [inline sme_finance_application_schema/entity_v1]: uhm, this departs from sme_v5, doesn't it? If that's the case, the sme_v5_and_contact_to_finance_application_v3 is broken

Comment (nesaro) [inline sme_finance_application_schema/entity_v1]: Confirmed, percentage in sme_v5

File: sme_finance_application_schema/entity_v1 [hunk 2]
```diff
@@ -104,9 +103,8 @@
         },
         "purchase_orders": {
             "title":"Purchase Orders",
-            "description":"The proportion of purchase orders the SME receives, expressed as a percentage",
+            "description":"The monthly value of purchase orders the SME receives, in GBP",
```
Comment (nesaro) [inline sme_finance_application_schema/entity_v1]: percentage in sme_v5

**Comment by nesaro** [comment]
This needs alignment with sme, so let's wait until sme_v6

**Comment by cybojenix** [comment]
stale

**Pull request closed without merging**